modules/nw-ovn-kubernetes-examine-database-contents-ref.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: REFERENCE
+[id="nw-examining-ovn-database-contents_{context}"]
+= Examining OVN database contents with ovn-nbctl and ovn-sbctl utilities
+
+OVN components may be examined with the `ovn-nbctl` and `ovn-sbctl` utilities.
+
+To run OVN commands, find the `ovnkube-master-xxxx` raft leader pod. See _Listing OVN northbound database contents with ovn-kubernetes_ and _Listing OVN southbound database contents with ovn-kubernetes_ for guidance on finding the raft leader for the northbound and southbound database.
+
+When you have identified the raft leader for the northbound database run the following command to show the northbound database contents:
+
+[source,terminal]
+----
+$ oc exec -n openshift-ovn-kubernetes -it ovnkube-master-gt4ms -c northd -- ovn-nbctl show
+----
+
+When you have identified the raft leader for the southbound database run the following command to show the southbound database contents:
+
+[source,terminal]
+----
+$ oc exec -n openshift-ovn-kubernetes -it ovnkube-master-mk6p6 -c northd -- ovn-sbctl show
+----
+
+Additional command line arguments are described in this table:
+
+.Examining database contents
+[cols="30%,70%",options="header"]
+|===
+| Argument | Description
+
+| `ovn-nbctl show`
+|An overview of the northbound database contents.
+
+| `ovn-nbctl show [switch \| router]`
```

Comment:
I have not seen this format with brackets and a pipe as a variable in a table heading. I think that this should be our standard user supplied variable of `<switch_or_router>`. The same comment applies throughout the table where brackets are used.